app/javascript/chat/chat.jsx

```diff
@@ -1314,21 +1302,24 @@ export class Chat extends Component {
           {invitesButton}
           {joiningRequestButton}
           <div className="chat__channeltypefilter">
-            {this.renderChannelFilterButton(
-              'all',
-              'all',
-              state.channelTypeFilter,
-            )}
-            {this.renderChannelFilterButton(
-              'direct',
-              'direct',
-              state.channelTypeFilter,
-            )}
-            {this.renderChannelFilterButton(
-              'invite_only',
-              'group',
-              state.channelTypeFilter,
-            )}
+            <ChannelFilterButton
+              type={'all'}
```

Comment:
(nitpick/not blocking) - we don't need the `{}` in these cases - same for all the string props below
```suggestion
              type="all"
```

Reply:
Weird that eslint/prettier didn't clean this up.

Reply:
@nickytonline looks like there's an eslint rule that can be enabled for it - I'll take a look and we can think about adding it in